recipe/bld.bat

```diff
@@ -1,7 +1,7 @@
-rm -rf C:/hostedtoolcache/windows/Python
+@echo on
 
-mkdir build
-cd build
+rem this is not needed anymore?
+rem rm -rf C:/hostedtoolcache/windows/Python
```

Comment:
looks like this is still needed:

> -- Shiboken6Config: Using default python: .cp310-win_amd64
> CMake Error at D:/bld/freecad_1730024460162/_h_env/Library/lib/cmake/Shiboken6/ShibokenHelpers.cmake:474 (message):
> -- Found Python: C:/hostedtoolcache/windows/Python/3.12.7/x64/python3.exe (found suitable version "3.12.7", minimum required is "3") found components: Interpreter Development Development.Module Development.Embed
>   The detected Python minor version is not compatible with the Python minor

although I would put this on an if statement to detect if running in azure if possible